code/game/machinery/computer/sm_monitor.dm

```diff
@@ -0,0 +1,142 @@
+/obj/machinery/computer/sm_monitor
+	name = "supermatter monitoring console"
+	desc = "Used to monitor supermatter shards."
+	icon_keyboard = "power_key"
+	icon_screen = "smmon_0"
+	circuit = /obj/item/circuitboard/sm_monitor
+	light_color = LIGHT_COLOR_YELLOW
+	/// Cache-list of all supermatter shards
+	var/list/supermatters = list()
```

Comment:
[Lazy list](https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md#startupruntime-tradeoffs-with-lists-and-the-hidden-init-proc)?